packages/params/src/activePreset.ts

```diff
@@ -60,4 +65,57 @@ export const {
   INACTIVITY_PENALTY_QUOTIENT_ALTAIR,
   MIN_SLASHING_PENALTY_QUOTIENT_ALTAIR,
   PROPORTIONAL_SLASHING_MULTIPLIER_ALTAIR,
-} = preset;
+} = presets[ACTIVE_PRESET];
+
+/**
+ * Override the active preset
+ *
+ * WARNING: Lodestar libraries rely on preset values being _constant_, so the active preset must be set _before_ loading any other lodestar libraries.
+ *
+ * Only call this function if you _really_ know what you are doing.
+ */
+export function setActivePreset(presetName: PresetName): void {
+  ACTIVE_PRESET = parsePresetName(presetName);
+  const preset = presets[ACTIVE_PRESET];
+
+  // Unfortunately, there is way to dynamically update all of these variables
+  // so they must be updated individually and explicitly
+  MAX_COMMITTEES_PER_SLOT = preset.MAX_COMMITTEES_PER_SLOT;
```

Comment:
Would doing something like 
```typescript
Object.entries(preset).forEach((k, v) => globalThis[k] = v
```

or some variant of that be dangerous terrible idea? :laughing:

Reply:
There's no way to access the module scope (as far as I know). `globalThis` isn't the same thing.